ansi/sixel/util.go

```diff
@@ -0,0 +1,8 @@
+package sixel
+
+func max(a, b int) int {
```

Comment:
this function seems quite simple do we need this? (doesn't exist in std?)

and I wonder if actually make sense to have in util.go if isn't used in more than 1 file

Reply:
it does in go1.20+ I think